scripts/production/run_healthchecks.pl

```diff
@@ -0,0 +1,211 @@
+my ($repair, $ensj_testrunner, $ensj_json_config);
+
+# Default values
+$master_db = 'compara_master';
+
+# Arguments parsing
+GetOptions(
+    'url=s'                         => \$url,
+    'reg_conf|regfile|reg_file=s'   => \$reg_conf,
+    'reg_type=s'                    => \$reg_type,
+    'reg_alias|regname|reg_name=s'  => \$reg_alias,
+    'master_db|compara_master=s'    => \$master_db,
+    'ensj-testrunner=s'             => \$ensj_testrunner,
+    'ensj-json-config=s'            => \$ensj_json_config,
+    'repair'                        => \$repair,
+);
+
+unless ($url or ($reg_conf and $reg_alias)) {
+    print "\nNeither --url nor --reg_conf and --reg_alias is not defined. The URL should be something like mysql://ensro\@compara1:3306/kb3_ensembl_compara_59\nEXIT 2\n\n";
+    exit 2;
+}
+
+if ($reg_conf) {
+    Bio::EnsEMBL::Registry->load_all($reg_conf);
+}
+my $dba = $reg_alias
+    ? Bio::EnsEMBL::Registry->get_DBAdaptor( $reg_alias, $reg_type || 'compara' )
+    : Bio::EnsEMBL::Compara::DBSQL::DBAdaptor->new( -URL => $url );
+
+my $division = $dba->get_division;
+
+unless ($ensj_testrunner) {
+    die "Need to give the --ensj-testrunner option or set the ENSEMBL_CVS_ROOT_DIR environment variable to use the default" unless $ENV{ENSEMBL_CVS_ROOT_DIR};
+    $ensj_testrunner = $ENV{ENSEMBL_CVS_ROOT_DIR} . '/ensj-healthcheck/run-configurable-testrunner.sh';
+}
+die "'$ensj_testrunner' is not a valid executable" unless -x $ensj_testrunner;
+
+unless ($ensj_json_config) {
+    die "Need to give the --ensj-config option or set the ENSEMBL_CVS_ROOT_DIR environment variable to use the default" unless $ENV{ENSEMBL_CVS_ROOT_DIR};
+    $ensj_json_config = $ENV{ENSEMBL_CVS_ROOT_DIR} . "/ensembl-compara/conf/$division/ensj-healthcheck.json";
+}
+die "'$ensj_json_config' is not a valid file" unless -e $ensj_json_config;
+my $ensj_config = decode_json(slurp($ensj_json_config));
+
+
+# Common parameters
+my @params = (
+    '--host'    => $dba->dbc->host,
+    '--port'    => $dba->dbc->port,
+    '--driver'  => 'org.gjt.mm.mysql.Driver',
+    '--release' => $ENV{CURR_ENSEMBL_RELEASE} || software_version(),
+    '--test_databases'  => $dba->dbc->dbname,
+);
+
+# RO or RW user depending on the --repair option
+if ($repair) {
+    push @params, (
+        '--user'    =>  Bio::EnsEMBL::Compara::Utils::Registry::get_rw_user($dba->dbc->host),
+        '--password'=>  Bio::EnsEMBL::Compara::Utils::Registry::get_rw_pass($dba->dbc->host),
+        '--repair'  => 1,
+    );
+} else {
+    push @params, (
+        '--user'    =>  'ensro',
+    );
+}
+
+# Add the master database if one is available (otherwise defaults to ensj-healthcheck name matching)
+if ($master_db) {
+    my $master_dba = Bio::EnsEMBL::Compara::DBSQL::DBAdaptor->go_figure_compara_dba($master_db);
+    die "Cannot find a database with the alias '$master_db' in the Registry" unless $master_dba;
+    push @params, (
+        '--compara_master.database', $master_dba->dbc->dbname,
+    );
```

Comment:
Just a picky comment: naming variables `master_db` and `master_dba` might lead to hard-to-find bugs if you misspell the latter, i.e. by mistake you don't add the last `a`.

Reply:
I don't agree. This is how I always name my variables and have never struggled to find a bug caused by it.

Reply:
Yes, it's quite standard to name the variables _db, _dba and _dbc